funcs.go

```diff
@@ -227,7 +227,7 @@ func filterField(fnName string, obj interface{}, field, val string, cmp func(str
 
 		f := findField(fieldPath, v)
 
-		strVal := fmt.Sprintf("%v", f.Interface())
+		strVal := fmt.Sprintf("%#v", f.Interface())
```

Comment:
The problem with changing this is that we use the default representation of a type, as provided by #v, for comparisons when we filter.  If we switch from the default to the internal representation, users of the filter function would also need to quote their strings, which you'd need to escape in the script, e.g.,

'{{filter . "Ticker" "\"BCOM.L\""}}'

This is why the unit tests are failing.

We could leave the %v format specifier in filter and change table and select, but this would be confusing if you were relying on table to figure out what values to filter on, as they would be formatted differently to what filter selects.